applications/CompressiblePotentialFlowApplication/tests/potential_flow_test_factory.py

```diff
@@ -78,6 +78,39 @@ def test_Naca0012SmallCompressible(self):
             for file_name in os.listdir():
                 if file_name.endswith(".time"):
                     kratos_utilities.DeleteFileIfExisting(file_name)
+
+    def test_Naca0012SmallTransonic(self):
```

Comment:
This test has to be skipped if the `EigenSolversApplication` is not available.

Reply:
To do this, you need to firstly check if the application is available (this can be done with `CheckIfApplicationsAvailable` method in the  `kratos_utilities.py`) and with the `Skip` test specifier (you can find examples in other applications).